Cargo.toml

```diff
@@ -133,6 +136,10 @@ metrics = [
     "influxdb",
     "dep:chrono",
 ]
+poi = [
```

Comment:
Should this be default?

Reply:
I think there's no real reason why it couldn't tbh. Wdyt @grtlr ?

Reply:
Yeah, let's make it default 👍

Reply:
Done